test/unit/main_test.ts

```diff
@@ -80,6 +80,8 @@ describe('Visitor pattern', () => {
   it('renames identifiers of property declarations/assignments', () => {
     expectTranslate('var foo = { bar: { baz: 12; } }; foo.bar.baz;')
       .to.equal('var foo = { $: { _: 12; } }; foo.$._;');
+    expectTranslate('var x = "something"; var foo = { bar: { baz: x; } }; foo.bar.baz;')
```

Comment:
what was the output before this change? The change makes it look like we are renaming less, but the test looks like we want to rename more.